docs/howto/how-to-make-api-tool/api-tool-guide.md

```diff
@@ -0,0 +1,341 @@
+        NBA_SEASON = 2016
+        lbj_stats = get_postseason_stats(LEBRON_JAMES_PLAYER_ID, NBA_SEASON)
+        table = get_pyarrow_table_from_stats(lbj_stats)
+        output_table = compute_aggregate_output(table)
+        self.provider.write_to_anchor("Output", output_table)
+        self.provider.io.info("APITool tool done.")
+```
+
+And now you're done with writing the code!
+## Package into a YXI 
+---
+Back in our plugin workspace, run the `ayx_plugin_cli create-yxi` command which bundles all the plugins in the workspace into a `.yxi` archive. It should look something like this:
+```powershell
+$ ayx_plugin_cli create-yxi
+[Creating YXI] started
+[Creating YXI] -- generate_config_files:generate_config_xml
+[Creating YXI] -- generate_config_files:generate_tool_config_xml
+[Creating YXI] .  generate_config_files:generate_manifest_jsons
+[Creating YXI] Generating manifest.json for APItool...
+[Creating YXI] Done!
+[Creating YXI] .  generate_artifact:build_artifacts
+[Creating YXI] Creating APItool.yxi...
+[Creating YXI] Creating shiv artifact...
+[Creating YXI] [Installing local dependencies]: python -m pip install -r requirements-local.txt --upgrade --target ~\sdk-api-tool\.ayx_cli.cache\dist
+[Creating YXI] [Compiling shiv artifact]: shiv --compile-pyc --reproducible --extend-pythonpath --site-packages ~\sdk-api-tool\.ayx_cli.cache\dist -o ~\sdk-api-tool\main.pyz -e ayx_python_sdk.providers.amp_provider.__main__:main
+[Creating YXI] Created shiv artifact at: ~\sdk-api-tool\main.pyz
+[Creating YXI] .  create_yxi:create_yxi
+[Creating YXI] finished
+```
+## Run Against Test Client
+---
+After creating the `.yxi` and before installing into Designer, we can first run our plugin against the Test Client to make sure there are no errors and that our plugin will run as expected. 
+Doing so speeds up development time, since we don't have to package and install the `.yxi` into Designer and create a new workflow everytime we make changes to our plugin code. 
+
+Run the `.yxi` against the Test Client using the command:
+
+```powershell
+$ ayx-sdk-cli.exe plugin run APITool -o output.json
+```
+
+This will output the results to a file called `output.json`.[^1] 
+
+## Installing and Runnning in Designer
+---
+In this section we will go over the two ways to install the plugin into Designer.
+### Method 1
+After you've finished creating a `.yxi`, you can install it into Designer by double-clicking on the `.yxi`, which will open Designer and prompt you to install the package in a new dialog box. It will look something like:
+
+![YXI Install Dialog](./assets/install-yxi-dialog.png)
+
+Once it finishes installing, the plugin can be found under the `Python SDK Examples` tab.[^2]
+
+### Method 2
+We can also create the yxi _**and**_ install it all at once by running the `ayx_plugin_cli designer-install` command. Choose the install option that matches your Designer install. Typically, this will be the `user` install option. 
```

Comment:
@ayx-orion Update to:

You can also create the YXI _**and**_ install it all at once via the `ayx_plugin_cli designer-install` command. Choose the install option that matches your Designer install. Typically, this is the `user` install option.